config/core/300-resources/service.yaml

```diff
@@ -58,6 +58,86 @@ spec:
+                        that the latest ready Revision of the Configuration should be used
+                        for this traffic target. When provided latestRevision MUST be true
+                        if revisionName is empty, and it MUST be false when revisionName is non-empty.
+                    tag:
+                      type: string
+                      description: |
+                        Tag is optionally used to expose a dedicated URL for
+                        referencing this target exclusively. The dedicated URL MUST include
+                        in it the string provided by tag.
+                    percent:
+                      type: integer
+                      description: |
+                        The percentage of requests which should be allocated
+                        from the main Route domain name to the specified `revisionName` or
+                        `configurationName`.
+                        All `percent` values in `traffic` MUST sum to 100.
+                      minimum: 0
+                      maximum: 100
+                    url:
+                      type: object
+                      description: |
+                        The URL at which the tag endpoint is reachable. It
+                        MUST not be taken as input, and is only provided on Status.
+                      x-kubernetes-preserve-unknown-fields: true
+                      properties:
+                        Scheme:
+                          type: string
+                        Opaque:
+                          type: string
+                          description: encoded opaque data
+                        User:
+                          type: object
+                          description: username and password information
+                          properties:
+                            username:
+                              type: string
+                            password:
+                              type: string
+                            passwordSet:
+                              type: boolean
+                        Host:
+                          type: string
+                          description: host or host:port
+                        Path:
+                          type: string
+                          description: path (relative paths may omit leading slash)
+                        RawPath:
+                          type: string
+                          description: encoded path hint (see EscapedPath method)
+                        ForceQuery:
+                          type: boolean
+                          description: append a query ('?') even if RawQuery is empty
+                        RawQuery:
+                          type: string
+                          description: encoded query values, without '?'
+                        Fragment:
+                          type: string
+                          description: fragment for references, without '#'
```

Comment:
This isn't accurate. You'll have to look at how the struct is serialized

Reply:
Actually the `url` property I believe is only present in the object's status block so I believe this whole thing can be removed.

Reply:
The struct for `TrafficTarget` has 
    URL *apis.URL `json:"url,omitempty"`
I used the fields for `url.URL`. 

Can you point me to where it is serialized. I'm confused if the type is URL and the schema has the details of URL, what is wrong. I'm not disagreeing, I'm just confused.

Reply:
Just saw the comment to remove it. I can do that if you want.

Reply:
We should be able to remove it from the spec

Reply:
ok, I'll remove it